Add voice toggle and onesync support

nathanctech opened the pull request:
This PR does a couple things:

1. Updates MAX_PLAYERS to 128 which makes it onesync compatible.
2. Adds a "Enable Voice Control" toggle to voice options. When turned off, it prevents Lambda from applying its voice settings. This will prevent some voice issues if used with other menus (like vMenu).
3. Updated scripthookv library. Not much used here, but might as well.
4. Updated toolset to v141 and VC++17.

I removed the `trigger_event` reference since I could not get it to compile otherwise.

nathanctech committed:
update toolset to v141 (vs2017)
Add voice control on/off for server menu compatibility

nathanctech committed:
Merge branch 'master' into onesynccompat

nathanctech committed:
Remove trigger_event due to compilation issues
Update scripthookv library
Add a couple missing references for vc++17

nathanctech committed:
fix compilation issue (oops)
move enums.h to the right place

gon453 commented:
Thanks for the update
I have a little problem when I compile your code
msgpack.hpp: No such file or directory, LambdaMenu c: \ users \ usernk \ desktop \ project-lambdamenu-master \ lambdamenu \ event_handler.h 1
tell me how to fix it please

nathanctech committed:
Update Lambda Menu for OneSync, add voice feature toggle.
noclip: wait to prevent main thread freeze
script: update to 128 players
script: add voice control toggle to prevent interference with other menus that also set voice options.
LambdaMenu: update to v142 build tools with latest VC++ draft.
